UnityProject/Assets/Scripts/PlayGroups/Input/InputController.cs

```diff
@@ -225,6 +225,16 @@ public bool Interact(Transform _transform, Vector3 position)
 					if (objectBehaviour.visibleState)
 					{
 						inputTrigger.Trigger(position);
+
+						//FIXME currently input controller only uses the first InputTrigger found on an object
+						/////// some objects have more then 1 input trigger, like players for example
+						/////// below is a solution that should be removed when InputController is refactored
+						/////// to support multiple InputTriggers on the target object
+						if (inputTrigger.gameObject.layer == 8) {
```

Comment:
Magic numbers.

Reply:
Please keep comments clear and ontopic.
None memming or alike in reviews please.

Reply:
Magic numbers is not a meme.
It's really unclear what "layer 8" represents

Reply:
In that case he could have responded with:
"It's really unclear what "layer 8" represents" instead.

Reply:
If you know (and you should) what a magic number is you really don't need me to be specific here.

Reply:
Not every contributor may know.
Please be more specific next time.